test/unit/io_managers_test.py

```diff
@@ -167,3 +172,76 @@ def test_missing_schema_error(sqlite_io_manager_fixture):
     output_context = build_output_context(asset_key=AssetKey(asset_key))
     with pytest.raises(RuntimeError):
         manager.handle_output(output_context, venue)
+
+
+def test_exclude_tables_param(tmp_path):
+    """Test ability to exclude tables from being created in the database."""
+    md = MetaData()
+    artist = Table(  # noqa: F841
+        "artist",
+        md,
+        Column("artistid", Integer, primary_key=True),
+        Column("artistname", String(16), nullable=False),
+    )
+    track = Table(  # noqa: F841
+        "track",
+        md,
+        Column("trackid", Integer, primary_key=True),
+        Column("trackname", String(16), nullable=False),
+        Column("trackartist", Integer, ForeignKey("artist.artistid")),
+    )
+    io_manager = SQLiteIOManager(
+        base_dir=tmp_path, db_name="pudl", md=md, exclude_tables=("track")
+    )
+    with io_manager.engine.connect() as con:
+        table_names = pd.read_sql(
+            "SELECT name FROM sqlite_schema WHERE type ='table' AND name NOT LIKE 'sqlite_%';",
+            con,
+        )
+
+    # Make sure the track table is in the metadata object but not created in the database
```

Comment:
Are there places where this discrepancy between the metadata object and the DB might end up biting us unexpectedly?

Reply:
We'll have to filter out the view tables from the metadata object when checking for changes in the database #2331.

I think [we should probably pass a `Package`](https://github.com/catalyst-cooperative/pudl/pull/2459#discussion_r1150885180) object into the `SQLiteIOManager` so we don't have views table metadata in `self.md`

Reply:
This has been resolved by the changes in https://github.com/catalyst-cooperative/pudl/pull/2466 and https://github.com/catalyst-cooperative/pudl/pull/2459.